app/utils/publication-email.js

```diff
@@ -18,10 +18,21 @@ const footer = 'Met vriendelijke groet,\n'
   + 'publicatiesBS@vlaanderen.be\t\n'
   + 'Koolstraat 35, 1000 Brussel\t\n';
 
-function translationRequestEmail(params) {
-  const dueDate = params.dueDate ? moment(params.dueDate).format('DD-MM-YYYY') : '-';
+async function buildContactInformation(contactPersons) {
+  let message = '\n' + 'Contact personen:\t\n'
```

Comment:
Deze samenstelling schrijf je aan elkaar. Dit is eentje waar we regelmatig tegenlopen. Onze codebase is in het Engels, maar de taal van de gebruiker is Nederlands. In het Engels worden samenstellingen meestal los van elkaar geschreven, in het Nederlands aan elkaar :nerd_face:

Reply:
When grammar leaves ur brain haha